pkg/cloudprovider/aws/fleet/packing/instancetype.go

```diff
@@ -0,0 +1,159 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package packing
+
+import (
+	"fmt"
+	"sort"
+	"sync"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// TODO get this information from node-selector
+var (
+	instanceTypesAvailable = []*instanceType{
```

Comment:
Could you dynamically populate this instead of hard coding? Eg w/ https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstanceTypes.html ?

Reply:
Happy to punt on this to a future PR if this gets us off the ground

Reply:
yes, we will be getting this all programmatically by calling EC2 API / by using node-selector